nerfstudio/data/dataparsers/base_dataparser.py

```diff
@@ -62,6 +56,7 @@ class DataparserOutputs:
     scene_box: SceneBox = SceneBox()
     """Scene box of dataset. Used to bound the scene or provide the scene scale depending on model."""
     semantics: Optional[Semantics] = None
+
```

Comment:
You can remove this newline.

Reply:
> Thanks for the PR. I think it's fine to switch to a single semantics class as you suggest. The current code works for panoptic segmentation from [detectron2](https://github.com/facebookresearch/detectron2/tree/main/configs/COCO-PanopticSegmentation).
> 
> When thing semantics == 0, it is the stuff class. When stuff semantics == 0, it is the thing class. You can see this by looking at the `panoptic_classes.json` file.
> 
> ```json
> {
>     "stuff": [
>         "thing",
>         "banner",
>         "blanket",
>         ...
>     ],
>     ....
>     "thing": [
>         "stuff",
>         "person",
>         "bicycle",
>         ...
>     ],
>     ....
> }
> ```

Sorry Ethan I updated my original description. 
Basically it is not always the case that when stuff == 0, thing != 0.

Reply:
And I am confused about how to handle this case for the current friends dataset with the proposed changes

Reply:
For the friends dataset, I think we can just remove the stuff classes for now. Can you incorporate that change into this PR to avoid that code breaking?

Reply:
Thats basically what Ive already done so it should be good to go in that case!